tests/src/controller/avdeccControlledEntity_tests.cpp

```diff
@@ -599,27 +599,28 @@ class JobPusherObserver : public la::avdecc::controller::Controller::DefaultedOb
 {
 public:
 	JobPusherObserver(la::avdecc::ExecutorManager::ExecutorWrapper* executor)
-		: _executor(executor)
+		: _executor({ executor })
 	{
 	}
 	virtual void onEntityOnline(la::avdecc::controller::Controller const* const /* controller */, la::avdecc::controller::ControlledEntity const* const /* entity*/) noexcept override
 	{
 		_executor->pushJob(
-			[this]()
+			[]()
 			{
+
```

Comment:
This new line just appeared. Good thing anyway, please add a very small comment:
// Empty, just to force a job in the queue

And do the same for the other empty job. Thanks